### ISSUE TITLE
Fragment selector

### ISSUE THREAD

**Issue by sorchadubh** [tify-iiif-viewer/tify#160]
Are there any plans to implement fragment selector (eg as in Mirador)?

**Comment by t11r** [comment]
Yes, but I cannot give you a timeframe yet. Could you provide a real-world example for us to test against?

**Comment by sorchadubh** [comment]
Thank you! I've been watching wip here with great interest. 
I hope I have used the right term here. What I'm chasing is selection and outline of areas of full-text to match the image, as in https://catdev.marshlibrary.ie/manifests/test4.json (ignore any other infelicities in this one!).
![image](https://github.com/tify-iiif-viewer/tify/assets/2345917/2b5890f3-92cb-480e-9e1e-3164432872ed)

I prefer tify to Mirador because it is lightweight and much easier to integrate: wip but this is the integration to VuFind as it is atm 

![image](https://github.com/tify-iiif-viewer/tify/assets/2345917/f889da7f-2c2c-40fa-bafa-9f66dc315da3)

**Comment by t11r** [comment]
Thanks! We will consider this as one of our test cases.

**Comment by t11r** [comment]
Supported in the [latest release](https://github.com/tify-iiif-viewer/tify/tree/v0.32.0).
